importer/v8/importer.go

```diff
@@ -153,18 +157,36 @@ func (i *Importer) processDDL(scanner *bufio.Scanner) {
 		if strings.TrimSpace(line) == "" {
 			continue
 		}
-		i.queryExecutor(line)
+
+		i.createDatabaseQuery = line
```

Comment:
This constantly sets `createDatabaseQuery` to the last line of the DDL section. Is that correct? Does `CREATE DATABASE` have to be the final line of the DDL preamble?

Also, whatever `line` is -- it was executed before being stored here; meaning at minimum, we will CREATE the database in the DDL section and then create the database for the override; that seems incorrect. If an override is specified, I wouldn't expect the database in the file to be created.